src/main/scala/com/cognite/sdk/scala/v1/resources/files.scala

```diff
@@ -24,24 +30,58 @@ class Files[F[_]](val requestSession: RequestSession[F])
 
   implicit val errorOrFileDecoder: Decoder[Either[CdpApiError, File]] =
     EitherDecoder.eitherDecoder[CdpApiError, File]
+
+  override def createOne(item: FileCreate): F[File] =
```

Comment:
Leave `createOne` for https://github.com/cognitedata/cognite-sdk-scala/issues/21